tests/Test/Krank/Checkers/IssueTrackerSpec.hs

```diff
@@ -0,0 +1,43 @@
+module Test.Krank.Checkers.IssueTrackerSpec (
+  spec
+  ) where
+
+import Test.Hspec
+
+import Text.Regex.Applicative ((=~))
+
+import Krank.Checkers.IssueTracker
+
+spec :: Spec
+spec = do
+  context "Test.Krank.Checkers.specIssueTracker" $ do
+    describe "#githubRE" $ do
+      it "handles full https url" $ do
+        "https://github.com/guibou/krank/issues/1" =~ githubRE `shouldBe` Just 1
```

Comment:
Cool. I'd see something such as

```haskell
$ do
   let matchGithubRe url = url =~ githubRE `shouldBe` Just 1
```

But that's fine as it is too.

Reply:
why? I'm not seeing the point of capturing that. Actually would that capture the spec or the match?